coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRuntime.java

```diff
@@ -578,7 +578,7 @@ class CoordinatorContext {
         /**
          * The current state.
          */
-        CoordinatorState state;
+        volatile CoordinatorState state;
```

Comment:
Is it necessary to backport this to active branches to prevent a potential race condition?